_build_cfg.yml

```diff
@@ -222,11 +222,13 @@ Topics:
     Dir: install
     Distros: openshift-origin,openshift-enterprise
     Topics:
-      - Name: Overview
-        File: index
+      - Name: Planning
+        File: planning
       - Name: Prerequisites
         File: prerequisites
-      - Name: RPM vs Containerized
+      - Name: Host Preparation
+        File: host_preparation
+      - Name: Containerized Services
```

Comment:
I like Containerized Components, so I'll change it to that. My main concern was moving away from the title "RPM vs Containerized", which isn't an accurate title description. (Though I've added some proper RPM vs Containered content in the Planning section)